data-egress.tf

```diff
@@ -238,7 +238,7 @@ resource "aws_dynamodb_table_item" "data_warehouse_config" {
 
   item = <<ITEM
   {
-    "source_prefix":                {"S":    "dwh/zip/*"},
+    "source_prefix":                {"S":    "dataegress/dwh/*"},
```

Comment:
This is a bit too generic for my liking, the other products have a folder name after dwh (or sas or ris) to denote what the data product we are sending is.

Reply:
i.e. this pattern would match all products we send to the datawarehouse, and if we left it as above. The next datawarehouse job would need to change this.